### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by goicoecheagabriel** [Ecwid/ecwid-js-sdk#3]
This can make ecwid a wonder!
Do we know when a new release is planned?
I will try it when I can manage the store.
-------****------------
¡Esto puede hacer que ecwid sea una maravilla!
¿Sabemos cuándo está prevista una nueva versión?
La probaré cuiando pueda gestionar la tienda.

**Comment by Erudition** [comment]
Agreed! Currently this is useless, have to do manual HTTP calls for everything
